docs/fundamentals/flow/client.md

```diff
@@ -193,7 +193,7 @@ f = (
 
 with f:  # Using it as a Context Manager will start the Flow
     client = Client(port=f.port)
-    docs = client.post(on='/', target_executor='barExecutor')
+    docs = client.post(on='/', target_executor='bar*')
     print(docs.texts)
 ```
 
```

Comment:
```suggestion
This will send the request to all Executors whose names start with 'bar', such as 'barExecutor'.
In the simplest case, you can specify a precise Executor name, and the request will be sent only to that single Executor.
```